wbooks/src/App/Auth/authentication.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+import { API } from './api';
+
+export const Auth = {
+  isLoggedIn() {
+    return (localStorage.getItem('token') !== null);
+  },
+  login(user, pass) {
+    return axios.post(API.baseURL + API.endpoint,{
+        email: user,
+        password: pass
+    })
+    .then((response) => {
+      localStorage.setItem('token',response.data.access_token);
+      return (response)
+    })
+    .catch((error) => {
+      return error.response.data;
+    });
```

Comment:
You can write this in one line:
```js
. catch((error) => error.response.data)
```
If you write an arrow function without curly braces '{}'  you don't need a `;` and it has an implicit return.